lib/src/rules/avoid_returning_null.dart

```diff
@@ -75,25 +75,25 @@ class _Visitor extends SimpleAstVisitor<void> {
 
   @override
   void visitFunctionExpression(FunctionExpression node) {
-    if (_isPrimitiveType(node.declaredElement.returnType)) {
+    if (_isPrimitiveType(node.declaredElement!.returnType)) {
       _visitFunctionBody(node.body);
     }
   }
 
   @override
   void visitMethodDeclaration(MethodDeclaration node) {
-    if (_isPrimitiveType(node.declaredElement.returnType)) {
+    if (_isPrimitiveType(node.declaredElement!.returnType)) {
       _visitFunctionBody(node.body);
     }
   }
 
-  void _visitFunctionBody(FunctionBody node) {
+  void _visitFunctionBody(FunctionBody? node) {
```

Comment:
`node` should be non-nullable. The analyzer APIs have been updated to disallow any function from having a null body. The compiler won't catch the change when you update to the next version of analyzer.